ext/rbs_extension/location.c

```diff
@@ -224,10 +227,16 @@ static VALUE rbs_new_location_from_loc_range(VALUE buffer, rbs_loc_range rg) {
   return obj;
 }
 
+static rbs_constant_id_t rbs_constant_pool_find_ruby_symbol(VALUE symbol) {
```

Comment:
So it's just `rbs_find_constant_id_from_ruby_symbol` renamed and moved to a different place?

If that's the case, can we not move the location of so it's easier to review?

Reply:
The diff was pretty messed up anyway, because every single line of it changed. So I just moved it closer to its point of use (there's only 1 caller)